holoviews/core/spaces.py

```diff
@@ -495,6 +495,13 @@ def __call__(self, *args, **kwargs):
         return ret
 
 
+class OverlayCallable(Callable):
+    """
+    A Callable subclass specifically meant to indicate that the Callable
+    represents an overlay operation between two objects.
+    """
```

Comment:
Why not make this an underscore attribute on ``Callable``? Then users have no reason to touch it or know about it?